docs/reference/usage-metrics.md

```diff
@@ -0,0 +1,48 @@
+---
+id: usage-metrics
+title: "Usage metrics"
+description: "Learn about the three main usage metrics that impact Camunda Platform 8 pricing."
+---
+
+There are three main usage metrics that have an impact on Camunda Platform 8 pricing. It is important to understand these metrics and how to retrieve them.
```

Comment:
> It is important to understand these metrics and how to retrieve them.

I can already answer this, but a user might as *why* it is important to understand these metrics. Could be as simple as adding a word or two for purpose.

Reply:
> It is important to understand these definitions, their impact on billing, and how to retrieve them.

Added in an incoming commit. WDYT?

Reply:
Awesome 🚀